client/source/components/RelatedProducts/CompareWindow.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CompareWindow = (props) => {
+  return (
+    <div>This Will Transform into a fancy shmancy modal window</div>
+  )
+};
+
+export default CompareWindow;
```

Comment:
All files should end in an empty line. This is a |-|istorical thing, to avoid bugs with older code that expect a certain not-visible End Of File character to appear imemdiately after a newline.